nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -56,23 +56,23 @@ def determine_subtype(cls, node: ov.Node) -> Optional[Type[OperatorMetatype]]:
 @OV_OPERATOR_METATYPES.register()
 class OVConvolutionMetatype(OVOpMetatype):
     name = 'ConvOp'
-    op_names = ['Convolution']
+    op_names = ['convolution']
```

Comment:
didn't mean renaming op_names to low-case names, I thought about implementation of handling the general case. It looks like these changes impacts all who use op_names via `get_all_aliases`. Taking into account the huge changes and impact I would recommend handle the softmax issue adding `softmax` with low-case to softmax metatype and file the issue to OpenVINO SoftMax documentation https://docs.openvino.ai/latest/openvino_docs_ops_activation_SoftMax_1.html

Reply:
Agree. Let's keep names as there was. We need just to update the `SoftMax -> Softmax` and create a bug for the corresponding team.

Reply:
Yes, you are right. Will update only Softmax